packages/ppx_deriving_yaml/ppx_deriving_yaml.0.2.2/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "Yaml PPX Deriver"
+description:
+  "Deriving conversion functions to and from yaml for your OCaml types."
+maintainer: ["patrick@sirref.org"]
+authors: ["Patrick Ferris"]
+license: "ISC"
+homepage: "https://github.com/patricoferris/ppx_deriving_yaml"
+bug-reports: "https://github.com/patricoferris/ppx_deriving_yaml/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "yaml"
+  "ppx_deriving"
+  "alcotest" {with-test}
+  "mdx" {with-test & >= "2.0.0"}
+  "ocaml" {>= "4.08.1"}
+  "ppxlib" {>= "0.25.0"}
+  "odoc" {with-doc}
+]
```

Comment:
```suggestion
]
conflicts: [
  "ocaml-option-bytecode-only"
]
```
For the moment we could solve it this way and release. What do you think?

Reply:
Thanks for the patience @mseri -- I've re-released 0.2.2 over at https://github.com/ocaml/opam-repository/pull/25072 where hopefully I have some dune checks to disable byte code MDX tests. Thanks for the help in getting a good release into opam, let's hope the tests are all green !